lib/syncUserFollows.js

```diff
@@ -79,7 +79,6 @@ function userFollows (groupId, userId) {
 
 							log.info({operation, subOp: 'newConceptsToFollow', userId, group: groupId, newConceptsCount: newConceptsToFollow.length});
 							const followProps = Object.assign({}, followedProperties);
-							followProps.asMemberOf = groupId;
```

Comment:
Sanity check please @dapenguin Removing this if we are no longer going to be making use of it anywhere.